Runtime/Core/InstalledJavaVersionCheck/InstalledJavaVersionProvider.cs

```diff
@@ -57,10 +57,11 @@ public CheckInstalledJavaVersionResponse CheckInstalledJavaVersion(CheckInstalle
             }
 
             // example output: java version "1.8.0"
-            string[] outputWords = processOutput
+            var versionLine = outputMatch.Groups[0].ToString();
+            string[] outputWords = versionLine
                 .Split(' ');
 
-            if (outputWords.Length < 3)
+            if (versionLine.Length < 3)
```

Comment:
Is this change intentional? Isn't versionLine always something like `java version 1.8.xxx` and always has length greater than 3?

Reply:
It was because outputWords would have been the entire output. But this whole part got removed in new revision.